quipucords/scanner/network/connect.py

```diff
@@ -336,10 +336,22 @@ def _connect(manager_interrupt,
 
         final_status = runner_obj.status
         if final_status != 'successful':
-            log_message = _construct_playbook_error_msg(final_status)
+            message = _construct_playbook_error_msg(final_status)
             if final_status not in ['unreachable', 'failed']:
-                scan_task.log_message(log_message)
-                raise AnsibleRunnerException(final_status)
+                if not scan_task.systems_scanned:
+                    log_message = 'Unexpected ansible status %s.  '\
```

Comment:
Also, I know I provided the text for these messages, but I hate them know.  Very relaxing drive home brought me inner peace.  Let's use this instead.

```
Unexpected ansible status %s.  Failing scan because there were no successful system connections.  Ansible error: %s
```